src/containers/App/App.js

```diff
@@ -184,10 +186,17 @@ export function App({
   });
 
   const showLoadingState = isPropertiesPlaceholder || isMessagesPlaceholder;
-  const isFetchingConfig = isFetchingProperties || isFetchingMessages;
-  const loadingConfigError = propertiesError || messagesError;
+  const isFetchingConfig =
+    isFetchingProperties || isFetchingMessages || isFetchingExtensions;
+  const loadingConfigError =
+    propertiesError || messagesError || extensionsError;
+
+  function fetchData() {
```

Comment:
@AlanGreene wasnt 100% sure if this it the correct approach here for fetching the data via websocket

Reply:
yes this is good for now, after all resources have been fully migrated to react-query we'll be centralising this using [`invalidateQueries`](https://react-query.tanstack.com/reference/QueryClient#queryclientinvalidatequeries) and letting react-query take care of refetching all active queries automatically.